### ISSUE TITLE
[bug] re-filed from old repository: Request queue was disposed

### ISSUE THREAD

**Issue by mscheong01** [asyncer-io/r2dbc-mysql#92]
**Describe the bug**
original issue: https://github.com/mirromutth/r2dbc-mysql/issues/241
Hi, there was this bug in the old implementation where a connection would hang for hours, stopping the thread, and then throw this error:
```
java.lang.IllegalStateException: Request queue was disposed 
at dev.miku.r2dbc.mysql.client.RequestQueue.requireDisposed(RequestQueue.java:150)
  at dev.miku.r2dbc.mysql.client.RequestQueue.dispose(RequestQueue.java:139)
  at dev.miku.r2dbc.mysql.client.MessageDuplexCodec.channelInactive(MessageDuplexCodec.java:131)
  at io.netty.channel.AbstractChannelHandlerContext.invokeChannelInactive(AbstractChannelHandlerContext.java:262)
  at io.netty.channel.AbstractChannelHandlerContext.invokeChannelInactive(AbstractChannelHandlerContext.java:248)
  at io.netty.channel.AbstractChannelHandlerContext.fireChannelInactive(AbstractChannelHandlerContext.java:241)
  at io.netty.handler.codec.ByteToMessageDecoder.channelInputClosed(ByteToMessageDecoder.java:389)
  at io.netty.handler.codec.ByteToMessageDecoder.channelInactive(ByteToMessageDecoder.java:354)
  at io.netty.channel.AbstractChannelHandlerContext.invokeChannelInactive(AbstractChannelHandlerContext.java:262)
  at io.netty.channel.AbstractChannelHandlerContext.invokeChannelInactive(AbstractChannelHandlerContext.java:248)
````
**To Reproduce**
I'm not sure what triggers this bug 😢 but there were multiple reports on the original issue 

----

could you confirm if this issue was handled when catching up to r2dbc 1.0.0?
Also, thanks a lot for taking over maintainership of this project and supporting r2dbc 1.0.0 🙇‍♂️

**Comment by jchrys** [comment]
Thanks @mscheong01  for reporting the issue. 
We will look into it.

**Comment by jchrys** [comment]
Hello, @mscheong01.

We've attempted to reproduce the issue you reported, but unfortunately, we were unable to do so. In order to help us better understand and address the problem, could you please provide a minimal (though not necessarily minimal) reproducible example along with relevant environment information? Details such as whether you are using r2dbc-pool, specific configurations, the Spring version, and any other pertinent information would be extremely helpful for us to identify the cause of the problem.

Thanks!

**Comment by jchrys** [comment]
cant' repro

**Comment by jchrys** [comment]
If there was an issue, it is expected to be resolved by #119.
